packages/facade/src/apis/sheet/f-range.ts

```diff
@@ -26,10 +27,13 @@ import type {
 import { ICommandService } from '@univerjs/core';
 import type { ISetStyleCommandParams } from '@univerjs/sheets';
 import { SetRangeValuesCommand, SetStyleCommand } from '@univerjs/sheets';
+import type { IStyleTypeValue } from '@univerjs/sheets/controllers/basic-worksheet.controller.js';
```

Comment:
Becare about the import path. If `IStyleTypeValue` is not exported from `@univerjs/sheets`, make it exported in the `index.ts` file.